client/doc.go

```diff
@@ -0,0 +1,42 @@
+/*
+Package client provides a library for all client-side CONIKS operations.
+
+Introduction
+
+One crucial component of the CONIKS key management system are the clients
+verifying the cryptographic proofs returned by the CONIKS server as part of
+the registration, lookup and monitoring protocols. client implements the
+operations performed by CONIKS clients during these protocols. This document
+outlines each of these protocols from a client perspective.
+
+Registration
+
+- The user registers her username with the CONIKS client.
+
+- The client generates a new public-private key pair for this username,
+and stores the keys on the device.
+
+- The client sends a registration request
+        reg_req = (username, key)
+to the server (may be validated by a registration bot; see
```

Comment:
Doesn't the client actually send its request to the `bot` instead of the coniks-/key-server? At least the code looks like it isn't possible that the client contacts the server directly for registration.